src/main/kotlin/com/back/domain/member/service/MemberService.kt

```
@@ -0,0 +1,256 @@
+            }
+
+        val profileUrl = member.profileUrl
+        if (profileUrl.isNullOrEmpty()) {
+            throw ServiceException(ResultCode.BAD_REQUEST.code(), "삭제할 프로필 이미지가 없습니다.")
+        }
+
+        try {
+            fileStorageService.deletePhysicalFile(profileUrl)
+            member.updateProfileUrl(null)
+            memberRepository.save(member)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)